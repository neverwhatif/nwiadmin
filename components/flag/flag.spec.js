import React from 'react';
import { shallow } from 'enzyme';

import Flag from './index';

describe('Flag', () => {
    const component = shallow(<Flag>Lorem</Flag>);

    it('should render', () => {
        expect(component).toMatchSnapshot();
    });
});
