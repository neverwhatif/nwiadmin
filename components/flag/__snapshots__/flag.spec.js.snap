// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Flag should render 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Flag
    isOffset={false}
  >
    Lorem
  </Flag>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": "Lorem",
      "className": "root",
    },
    "ref": null,
    "rendered": "Lorem",
    "type": "span",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "Lorem",
        "className": "root",
      },
      "ref": null,
      "rendered": "Lorem",
      "type": "span",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
